resources/_gen/assets/js/js/search.js_b4bb3e7b21b43e920b7c367ff787d383.content

```diff
@@ -27,7 +27,7 @@ limitations under the License.
                     }
 
                     var query = $(this).val();
-                    var searchPage = "/search/?q=" + query;
+                    var searchPage = "//localhost:1313/search/?q=" + query;
```

Comment:
the reference to localhost looks like you might have made to test locally. Is this added to the PR in error?

Reply:
Yes! Good catch. I thought I'd reverted that but it snuck through.

I'm putting a hold on this PR until I see if I can improve the search results too. But please do continue with the review, especially the UI changes.
/hold